decred/tests/unit/dcr/test_txscript.py

```diff
@@ -813,6 +813,492 @@ def ssgenMsgTxWrongFirstOut():
     )
 
 
+class TestSubsidyCache(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        chains.registerChain("dcr", None)
```

Comment:
How about translating to pytest while we're at it?

Reply:
Ah, yes, of course.